pkg/console/operator/operator.go

```diff
@@ -237,19 +202,18 @@ func (c *consoleOperator) deleteAllResources(cr *consolev1alpha1.Console) error
 
 // this may need to eventually live under each sync version, depending on if there is
 // custom sync logic
-func (c *consoleOperator) defaultConsole() *consolev1alpha1.Console {
+func (c *consoleOperator) defaultConsole() *consolev1.Console {
 	logrus.Info("creating console CR with default values")
-	return &consolev1alpha1.Console{
+
+	return &consolev1.Console{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      api.ResourceName,
 			Namespace: api.OpenShiftConsoleNamespace,
 		},
-		Spec: consolev1alpha1.ConsoleSpec{
-			OperatorSpec: operatorsv1alpha1.OperatorSpec{
+		Spec: consolev1.ConsoleSpec{
+			OperatorSpec: operatorv1.OperatorSpec{
 				// by default the console is managed
-				ManagementState: operatorsv1alpha1.Managed,
-				// if Verison is not 4.0.0 our reconcile loop will not pick it up
-				Version: "4.0.0",
```

Comment:
Don't we need to keep the version ?

Reply:
Nope.  Version in [v1alpha1 spec](https://github.com/openshift/api/blob/master/operator/v1alpha1/types.go#L34) is no longer in [v1 spec](https://github.com/openshift/api/blob/master/operator/v1/types.go#L43).  It is in the status, however. The "version availability" fields have also gone away.  

For this PR, I want to get the operator status working again.  I think dealing with Version is something that needs to come as a follow-on, certainly by the time we are talking about upgrades from 4.0 -> 4.1.

Reply:
Reason: I believe operators are just supposed to report version status, but aren't supposed to control it.  Upgrades to version are handled by higher level components (CVO).  If a new version of the operator is rolled out, it ought to then upgrade the resources it is responsible for (if needed).

@spadgett if there is anything to add to this train of thought.